gax/retry_policy.h

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019 Google Inc.  All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_API_CODEGEN_GAPIC_RETRY_POLICY_H_
+#define GOOGLE_API_CODEGEN_GAPIC_RETRY_POLICY_H_
+
+#include <chrono>
+#include <memory>
+
+namespace google {
+namespace gapic {
```

Comment:
What about having the namespace be just `::google::gax`? Open to discussion here, but that was my first thought.

Reply:
My initial thought process was that the generator may want to include headers from gax, but this isn't really correct: the generator *emits* code that uses gax headers but doesn't use them itself. Changing the namespace to `::google::gax` makes sense.